<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChartWidget</class>
 <widget class="QWidget" name="ChartWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>数据分析</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#ChartWidget {
    background-color: #f5f7fa;
}

QFrame#chartCard {
    background-color: white;
    border-radius: 12px;
    border: 1px solid #e4e7ed;
}

QLabel#titleLabel {
    font-size: 18px;
    font-weight: bold;
    color: #303133;
}

QLabel#cityLabel {
    font-size: 16px;
    color: #606266;
}

QComboBox {
    padding: 6px 12px;
    border: 1px solid #dcdfe6;
    border-radius: 4px;
    background-color: white;
    min-width: 120px;
}

QComboBox:hover {
    border-color: #c0c4cc;
}

QComboBox::drop-down {
    border: none;
    width: 20px;
}

QPushButton#refreshBtn {
    background-color: #409EFF;
    color: white;
    border: none;
    border-radius: 6px;
    padding: 8px 16px;
    font-size: 14px;
}

QPushButton#refreshBtn:hover {
    background-color: #66b1ff;
}

QTabWidget::pane {
    border: none;
    background-color: transparent;
}

QTabBar::tab {
    padding: 8px 16px;
    margin-right: 4px;
    border-radius: 4px;
    background-color: #f5f7fa;
    color: #606266;
}

QTabBar::tab:selected {
    background-color: #409EFF;
    color: white;
}

QTabBar::tab:hover:!selected {
    background-color: #e4e7ed;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>16</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="headerLayout">
     <item>
      <widget class="QLabel" name="cityLabel">
       <property name="text">
        <string>选择城市查看分析</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="headerSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="chartTypeCombo">
       <item>
        <property name="text">
         <string>温度趋势</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>湿度变化</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>风速分析</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>气压变化</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="refreshBtn">
       <property name="text">
        <string>刷新</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="hourlyTab">
      <attribute name="title">
       <string>24小时</string>
      </attribute>
      <layout class="QVBoxLayout" name="hourlyTabLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>8</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="hourlyChartCard">
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <layout class="QVBoxLayout" name="hourlyChartLayout">
          <property name="leftMargin">
           <number>16</number>
          </property>
          <property name="topMargin">
           <number>16</number>
          </property>
          <property name="rightMargin">
           <number>16</number>
          </property>
          <property name="bottomMargin">
           <number>16</number>
          </property>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="dailyTab">
      <attribute name="title">
       <string>7日趋势</string>
      </attribute>
      <layout class="QVBoxLayout" name="dailyTabLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>8</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="dailyChartCard">
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <layout class="QVBoxLayout" name="dailyChartLayout">
          <property name="leftMargin">
           <number>16</number>
          </property>
          <property name="topMargin">
           <number>16</number>
          </property>
          <property name="rightMargin">
           <number>16</number>
          </property>
          <property name="bottomMargin">
           <number>16</number>
          </property>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
